### PR TITLE
Update autoconf to v2.69 on CentOS 6 and RHEL 6

### PR DESCRIPTION
**Pull request by vsebe**
Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
@jdekonin please review

**Comment by vsebe** [comment]
@karianna - autoconf is an extensible package of M4 macros. It is used in the `configure` stage to generate the configuration scripts (e.g. generated-configure.sh).

**Comment by karianna** [comment]
Oh sorry - I was conflating with the default autoconf that we were obviously relying on and you explicitly installing a particular version (which is better)

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/autoconf/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  when:
+    - (ansible_distribution == "RedHat" or ansible_distribution == "CentOS")
+    - ansible_distribution_major_version == "6"
+    - ansible_architecture == "x86_64"
+  register: autoconf_version
+  tags: autoconf-2.69
+
+- name: Download Autoconf v2.69
+  get_url:
+    url: http://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/autoconf/tasks/main.yml]: Can we https here?

Comment (vsebe) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/autoconf/tasks/main.yml]: Yes, we can.

Comment (vsebe) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/autoconf/tasks/main.yml]: url changed

**Comment by karianna** [comment]
It has failed our linter

` 74:1      error    too many blank lines (1 > 0)  (empty-lines)` in the autocnof yaml file

**Comment by vsebe** [comment]
@karianna - fyi... extra line has been removed

**Pull request merged**